### PR TITLE
v1.6.0

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

**Review by lucacasonato — APPROVED** [review]

File: Releases.md [hunk 1]
```diff
@@ -6,6 +6,55 @@ https://github.com/denoland/deno/releases
...
 https://github.com/denoland/deno_install
 
+### 1.6.0 / 2020.12.08
+
+- BREAKING: Make "isolatedModules" setting non-configurable (#8482)
+- feat: Add mvp language server (#8515, #8651)
+- feat: deno compile (#8539, #8563, #8581)
+- feat: Update to TypeScript 4.1 (#7573)
+- feat: EventTarget signal support (#8616)
+- feat: Add cbreak option to Deno.setRaw (#8383)
```
Comment (ry) [inline Releases.md:16]: unstable feature

File: Releases.md [hunk 2]
```diff
@@ -6,6 +6,55 @@ https://github.com/denoland/deno/releases
...
+- feat: Add canary support to upgrade subcommand (#8476)
+- fix: "onload" event order (#8376)
+- fix: Add file URL support for Deno.readLink (#8423)
+- fix: Add hygiene pass to transpile pipeline (#8586)
+- fix: Add write permissions for unixpackets datagrams & unix socket (#8511)
+- fix: Higlight `async` and `of` in REPL (#8569)
```
Comment (ry) [inline Releases.md:22]: highlight

File: Releases.md [hunk 3]
```diff
@@ -6,6 +6,55 @@ https://github.com/denoland/deno/releases
...
+- feat: Add mvp language server (#8515, #8651)
+- feat: deno compile (#8539, #8563, #8581)
+- feat: Update to TypeScript 4.1 (#7573)
+- feat: EventTarget signal support (#8616)
+- feat: Add cbreak option to Deno.setRaw (#8383)
+- feat: Add canary support to upgrade subcommand (#8476)
+- fix: "onload" event order (#8376)
+- fix: Add file URL support for Deno.readLink (#8423)
+- fix: Add hygiene pass to transpile pipeline (#8586)
+- fix: Add write permissions for unixpackets datagrams & unix socket (#8511)
```
Comment (ry) [inline Releases.md:21]: require allow-write permissions for unixpackets datagrams & unix socket

**Review by ry — APPROVED** [review]
LGTM

**Pull request merged**